tests/benchmarks/inspect_edges.yml

```diff
@@ -1,9 +1,10 @@
-name: "UPDATE-BASELINE"
+name: "INSPECT-EDGES"
 remote:
   - setup: redisgraph-r5
   - type: oss-standalone
 dbconfig:
-  - dataset: "datasets/1M_edges.rdb"
+  - init_commands:
+    - '"GRAPH.QUERY" "g" "UNWIND range(1, 1000000) AS x CREATE (src:N {v: x}), (src)-[:R]->(:N)"'
```

Comment:
`"UNWIND range(1, 1000000) AS x CREATE (:N {v: x})-[:R]->(:N)"`
Do we clear the database we're running against before running each benchmark?

Reply:
yes. we ensure always the same start conditions.